Makefile

```diff
@@ -2,7 +2,8 @@ SHELL := /bin/bash
 VERSION := $(shell sed 's/^__version__ = "\(.*\)"/\1/' ./mlserver/version.py)
 IMAGE_NAME := seldonio/mlserver
 
-.PHONY: install-dev _generate generate run build push-test push test lint fmt version clean licenses
+.PHONY: install-dev _generate generate run build build-pypy build-docker\
```

Comment:
:question: I can't see `build-pypy` or `build-docker` here - have those changes not yet been pushed?